components/_patterns/04-templates/example-page/example-page.twig

```diff
@@ -22,11 +31,20 @@
         "#",
     },
   }
-}%}
+} %}
 
 {% include "@organisms/card-grid/card-grid.twig" with {
+  "grid_blockname": "card",
+  "grid_modifiers": {
```

Comment:
As promised, here's an example of how to write multiples in the array:
```
"grid_modifiers": ["divider", "columns-3"],
```

Fewer lines, easier to read.